metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -74,6 +79,70 @@ def test_raise_failure_flow(pytestconfig) -> None:
     return
 
 
+def validate_node_selector_term(node_selector_term: Dict) -> bool:
+    for affinity_match_expression in node_selector_term["matchExpressions"]:
+        if (
+            affinity_match_expression["key"] == "k8s.amazonaws.com/accelerator"
+            and affinity_match_expression["operator"] == "In"
+            and "nvidia-tesla-v100" in affinity_match_expression["values"]
+        ):
+            return True
+    return False
+
+
+def validate_toleration(toleration: Dict) -> bool:
```

Comment:
this function does not validate the toleration, it solely returns whether the toleration is found.

so it should be called `is_nvidia_accelerator_noschedule` or something of that nature.

Same with `validate_node_selector_term` which should be more of `exists_nvidia_acceleartor`